NAMESPACE

```diff
@@ -5,7 +5,6 @@ S3method(print,rcmdcheck)
 S3method(print,rcmdcheck_comparison)
 S3method(summary,rcmdcheck)
 S3method(summary,rcmdcheck_comparison)
-export(check_details)
```

Comment:
I can un-remove it so we don't break the revdeps, but I think it's better to have a stricter definition of the object itself and then just use that directly.